### ISSUE TITLE
DFSG freeness and Ubuntu Font License

### ISSUE THREAD

**Issue by medicalwei** [fonttools/ufoLib2#12]
Sorry for the license issue here.  During packaging ufoLib2 into Debian, we have to strip out Ubuntu font used in the test cases.  It is considered not a free software license in Debian (not DFSG-free) since it has issues about naming restrictions.  (Forcing derivative works to use "Ubuntu derivative X" on font name, instead of prevent from using the name like Reserved Font Name in SIL OFL, where latter is allowed in DFSG to prevent trademark issues.)

Source:
https://alioth-lists.debian.net/pipermail/pkg-fonts-devel/2011-April/006515.html

**Comment by medicalwei** [comment]
Yes, I am one of the packagers, and we do run test suites when available.

**Comment by anthrotype** [comment]
i don't think we need this specific ubuntu font for testing. we will just replace it with noto sans or similar.
